src/customize-pop/customize-desktop.md

```diff
@@ -0,0 +1,99 @@
+# Customizing the Desktop
+
+Change the function of the `SUPER` key, desktop backgrounds, workspace features, and placement and visibility of desktop components in `Settings` ➞ `Desktop`.
+
+- [`Desktop Options`](/customize-pop/customize-desktop.md#desktop-options) provides options to modify the `SUPER` key's function, enabling Hot Corner, and adding or positioning top bar elements.
+- [`Background` settings](/customize-pop/customize-desktop.md#background) allows users to set pre-installed wallpaper or user images as the background image.
+- [`Appearance` settings](/customize-pop/customize-desktop.md#appearance) provides the option to choose a dark or light theme that applies across the entire desktop environment.
+- [`Dock` settings](/customize-pop/customize-desktop.md#dock) provides option to modify Dock visibility, placement, size, and displayed icons.
+- [`Workspaces` settings](/customize-pop/customize-desktop.md#workspaces) includes options configure workspace positioning and behavior.
+
+---
+
+## Desktop Options
+
+Customize basic desktop elements in the `Desktop Options` tab.
+
+![General Settings](/images/customize-desktop/general-settings.png)
+
+| Setting | Function |
+|----------|----------|
+| Super Key Action | Assigns the `SUPER` key to start the Launcher, display Workspace navigation, or view installed applications. |
+| Hot Corner | Displays workspace navigation when the user moves the mouse cursor to the upper left corner of the screen. |
```

Comment:
Why was `Workspace navigation` capitalized on the previous line if it's not here?